XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/BaseHandController.cs

```diff
@@ -84,6 +93,18 @@ public void UpdateController(HandData handData)
                 MixedRealityToolkit.InputSystem?.RaiseSourcePoseChanged(InputSource, this, wristPose);
             }
 
+            if (lastIsInInputDownPose != IsInInputDownPose)
+            {
+                if (IsInInputDownPose && !lastIsInInputDownPose)
+                {
+                    MixedRealityToolkit.InputSystem?.RaiseOnInputDown(InputSource, ControllerHandedness, MixedRealityInputAction.None);
```

Comment:
I would like to see a way to assign these actions in the controller mapping window somehow.